cmd/demo/node.go

```diff
@@ -431,7 +448,7 @@ func (n *node) settle(p *peer) error {
 	}
 	p.ch.log.Debug("Removing channel")
 	p.ch = nil
-	fmt.Println("🏁 Settled channel with", p.alias)
+	fmt.Printf("\n🏁 Settled channel with %s. Final Balance: [My: %v Ξ, Peer: %v Ξ]\n", p.alias, finalBals[0], finalBals[1])
```

Comment:
Define `me := p.ch.Idx()` before `nil`ing `p.ch` and
```suggestion
	fmt.Printf("\n🏁 Settled channel with %s. Final Balance: [My: %v Ξ, Peer: %v Ξ]\n", p.alias, finalBals[me], finalBals[1-me])
```

Reply:
Ordering of balances is done by p.ch.GetBalances() call (#442). So always my balance is at 0 and peer balance at 1.